resources/istio/values.yaml

```diff
@@ -394,11 +379,9 @@ pilot:
   traceSampling: 100.0
   # Resources for a small pilot install
   resources:
-    limits:
-      cpu: 500m
-      memory: 2Gi
     requests:
-      memory: 512Mi
+      cpu: 500m
+      memory: 2048Mi
```

Comment:
where u overriding that?

Reply:
This is because we have included raw istio charts. This value is provided in default charts: https://github.com/istio/istio/blob/1.0.2/install/kubernetes/helm/istio/values.yaml#L383 . If This is overridden using  installer overrides for [minikube](https://github.com/kyma-project/kyma/blob/fde018349b43782c8e1b13e4d8ebc8cacf4baed3/installation/resources/installer-config-local.yaml.tpl#L98) and [cluster](https://github.com/kyma-project/kyma/blob/fde018349b43782c8e1b13e4d8ebc8cacf4baed3/installation/resources/installer-config-cluster.yaml.tpl#L100).